lib/wasi/Cargo.toml

```diff
@@ -72,6 +70,7 @@ optional = true
 
 [target.'cfg(unix)'.dependencies]
 libc = { version = "^0.2", default-features = false }
+termios = { version = "0.3" }
```

Comment:
This should still be optional, no?
We don't need it by default, just if we want host terminal support.

Reply:
Well it can't be optional if we don't have a feature flag so its either one or the other